tests/features/producer/createDocumentReference-failure.feature

```diff
@@ -219,6 +219,45 @@ Feature: Producer - createDocumentReference - Failure Scenarios
       }
       """
 
+  Scenario: Invalid status
+    Given the application 'DataShare' (ID 'z00z-y11y-x22x') is registered to access the API
+    And the organisation 'X26' is authorised to access pointer types:
+      | system                 | value            |
+      | http://snomed.info/sct | 1363501000000100 |
+      | http://snomed.info/sct | 736253002        |
+    When producer 'X26' creates a DocumentReference with values:
+      | property  | value                          |
+      | subject   | 9999999999                     |
+      | status    | current                        |
```

Comment:
If the status is current, why would it fail? 🤔  Does it have a whitespace character or something?